src/options.rs

```diff
@@ -383,6 +386,27 @@ impl Options {
         self
     }
 
+    /// Select option to enable reconnect with backoff
+    /// on first failed connection attempt.
+    /// The reconnect logic with `max_reconnects` and the
+    /// `reconnect_delay_callback` will be specified the same
+    /// as before but will be invoked on the first failed
+    /// connection attempt.
+    ///
+    /// # Example
```

Comment:
Copy pasted example from async code.
This should contain sync example.

Reply:
My bad - updated.